docs/README.md

```diff
@@ -1,6 +1,11 @@
-# Duke
+# Upcycle
 
-{Give product intro here}
+Hi there, 
+
+Welcome to Upcycle - a perfect desktop app dedicate for managing rental businesses. The nature of these businesses
+requires to deal with tons of data; therefore, noting down all information by notes and pen is not ideal for managers.
+It is developed for rental business managers, who can type fast to efficiently keep track all of their customers, items,
+and transactions via Command Line Interface with a single line of command.
 
 Useful links:
 * [User Guide](UserGuide.md)
```

Comment:
Ohhhhh not return again leh
![8](https://user-images.githubusercontent.com/71815678/198498604-4e09e25f-7d16-4bce-bfa4-6898b96db555.png)